.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,161 @@
+---
+name: Release issue template
+about: Tracking issue for new releases
+title: Cumulus {{ env.VERSION }} Release checklist
+---
+# Release Checklist
+
+### Runtime Releases
+
+These checks should be performed on the codebase.
+
+- [ ] Verify [`spec_version`](#spec-version) has been incremented since the
+    last release for any native runtimes from any existing use on public
+    (non-private/test) networks.
+- [ ] Verify previously [completed migrations](#old-migrations-removed) are
+    removed for any public (non-private/test) networks. @apopiak
+  - No migrations added in the last release that would need to be removed.
+- [ ] Verify pallet and [extrinsic ordering](#extrinsic-ordering) as well as `SignedExtension`s have stayed
+    the same. Bump `transaction_version` if not. @NachoPal
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted for
+    [proxy filters](#proxy-filtering). @apopiak
+  - No new extrinsics.
+- [ ] Verify [benchmarks](#benchmarks) have been updated for any modified
+    runtime logic. @NachoPal
+  - [ ] Verify the weights are up-to-date. @NachoPal
+- [ ] Verify that the various pieces of XCM config are sane. @apopiak
+
+The following checks can be performed after we have forked off to the release-
+candidate branch or started an additional release candidate branch (rc-2, rc-3, etc)
+
+- [ ] Verify [new migrations](#new-migrations) complete successfully, and the
+    runtime state is correctly updated for any public (non-private/test)
+    networks.
+- [ ] Verify [Polkadot JS API](#polkadot-js) are up to date with the latest
+    runtime changes.
+- [ ] Push runtime upgrade to Westmint and verify network stability.
+
+### Initial Statemint Release
+On top of the regular runtime release process, this release will also need to cover the following points:
+- [ ] genesis configuration for Statemint @apopiak
+- [ ] determine `setStorage` call for governance to set the initial state Statemint should have after the upgrade (replacing the usual genesis process, but making use of the genesis config) @apopiak
+- [ ] create adusted chain spec that will be used for syncing Statemint @NachoPal
+- [ ] test the upgrade process (including `setStorage` calls) @apopiak
```

Comment:
I don't think this should be part of the general release notes, this is a one-time thing
In fact I should remove them from the v6 release